netket/vqs/base.py

```diff
@@ -181,7 +182,7 @@ def expect_and_grad(
         Ô: AbstractOperator,
         *,
         mutable: Optional[Any] = None,
-        is_hermitian: Optional[bool] = None,
+        covariance_formula: Optional[bool] = None,
```

Comment:
maybe can we rename this to ```use_covariance```or similar? otherwise this would make me think you are expecting a formula somehow and not a bool

Reply:
good point

Reply:
also, is there an advantage in using these custom bool types for dispatching instead of doing an if? (maybe there is a slight running time advantage but I am not sure this is measurable)